app/controllers/orders/detail.js

```diff
@@ -2,14 +2,21 @@ import Ember from "ember";
 import config from "../../config/environment";
 import AjaxPromise from "stock/utils/ajax-promise";
 import GoodcityController from "../goodcity_controller";
+import SearchMixin from "stock/mixins/search_resource";
 import _ from "lodash";
 const { getOwner } = Ember;
 
-export default GoodcityController.extend({
+export default GoodcityController.extend(SearchMixin, {
+  autoLoad: true,
+  /**
```

Comment:
I've added JSDoc to the project.

So commenting something with `/**` as a start will get captured in the doc, but the documentation tool won't know to which module it belong.

So options are:
* Document the controller itself (there are examples in the project)
* Simply start the comment using `/*` or `//`

Note: Documenting Controllers, although not a bad thing, is definitely not the most useful thing to document generally, since they're not meant to be re-used